frontend/src/Wrapper.tsx

```diff
@@ -175,6 +178,10 @@ export default function Wrapper({ title, page, children }: WrapperProps): ReactE
     },
   })
 
+  function displayBetaTag() {
```

Comment:
Is it better if you do:
```
const betaAdornment = <Box component="span" sx={{ marginLeft: 1, color: '#cecece', fontSize: 15 }}>beta</span>
```
Then you could use it below like:
```
              <Link
                href='/'
                color='inherit'
                underline='none'
                style={{ color: 'inherit', textDecoration: 'inherit', fontSize: '1.25rem', fontWeight: 500 }}
              >
                Bailo
                {router.asPath.includes('/beta') && betaAdornment}
              </Link>
```
I've renamed to betaAdornment because tags are a different thing within Bailo and it's worth avoiding any confusion.